t/lib/server.lua

```diff
@@ -269,6 +278,7 @@ function _M.go()
         return ngx.exit(404)
     end
 
+    inject_headers()
```

Comment:
Why we need change header key `resp-x` header to "x"?

Reply:
@ShiningRush 
It's for test case to control the response header. Note that the type of headers are different.

Reply:
Well, got it.